specification/communication/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to the root directory of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-communication"
+  output-folder: "$(typescript-sdks-folder)/sdk/communication/arm-communication"
```

Comment:
I just found that the folder https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/communication is not empty, Are those the communication data-plane SDKs?

Reply:
Correct. Those are the data-plane SDKs.